app/products/calls/components/captions.tsx

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {clearTimeout} from '@testing-library/react-native/build/helpers/timers';
```

Comment:
I believe this is wrong and not needed

Reply:
Oh dear, don't know how that got there. Thanks, removed.